src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.icon-small {
+  position: absolute;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  box-sizing: border-box;
+}
+
+input::placeholder {
+  /* font-family: Avenir, Arial, sans-serif;
+  font-size: 16px; */
+  color: #3d4e61;
+  /* font-style: inherit; */
+}
+
+input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.input-big {
+  height: 70px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  width: 100%;
+  padding-left: 62px;
+
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input-small {
+  height: 42px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  width: 100%;
+  padding-left: 33px;
+
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 14px;
+  color: #3d4e61;
+}
```

Comment:
if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.
 
![image](https://user-images.githubusercontent.com/61904995/222153847-711f2790-47cd-4d4b-bdf3-7e4d0fb083b1.png)